modular_skyrat/master_files/code/datums/quirks/neutral.dm

```diff
@@ -27,7 +27,8 @@
 				if (success)
 					break
 		equipped_items[item] = success
-	on_equip_items(equipped_items)
+	for (var/obj/item/equipped as anything in equipped_items)
+		on_equip_item(equipped, equipped_items[equipped])
```

Comment:
whats the difference between on_equip_items and on_equip_item... why would you rather have the singular and put a loop when I assume the plural does that.